src/components/migrationToV3.stories.mdx

```diff
@@ -539,11 +539,17 @@ In V3, we have improved the `type` for selected device in `MeetingManager` and c
 
 #### MeetingManager Properties Change
 
-| Before in 2.x                              | After in 3.x                                             |
```

Comment:
This was rendered differently in storybook due to the `|` coming before the table separator:
<img width="624" alt="Screen Shot 2022-04-19 at 3 46 34 PM" src="https://user-images.githubusercontent.com/61809205/164126093-0146c991-1c07-44d1-8727-53986885cdd9.png">

Reply:
Thanks for catching this!

We could escape `|` in this way, but it's too complex and I prefer your solution.

| Before in 2.x                                              | After in 3.x                                                             |
| ---------------------------------------------------------- | ------------------------------------------------------------------------ |
| <code>selectedAudioOutputDevice: string &#124; null</code> | <code>selectedAudioOutputDevice: string &#124; null</code>               |
| <code>selectedAudioInputDevice: string &#124; null</code>  | <code>selectedAudioInputDevice: AudioInputDevice &#124; undefined</code> |
| <code>selectedVideoInputDevice: string &#124; null</code>  | <code>selectedVideoInputDevice: VideoInputDevice &#124; undefined</code> |

Reply:
Cool dint knew about this! Well I kept it simple for now, can fix if this needs a revision :)

Reply:
No, it's good, no need to fix it